workflows/covidhub-import.wdl

```diff
@@ -0,0 +1,32 @@
+version 1.1
```

Comment:
why is there another wdl file?

Reply:
yeah, delete this. we'll just drop everything in one place for now and figure out how to move them later.